tensorly/contrib/decomposition/tests/test_mps_decomposition_cross.py

```diff
@@ -69,8 +66,9 @@ def test_tensor_train_cross_2():
         first_error_message += str(r_k) + " > " + str(rank[k+1])
         assert(r_k<=rank[k+1]), first_error_message
 
-@skip_if_jax
-@skip_if_tensorflow
+
+@skip_if_backend
+@pytest.mark.skipif(tl.get_backend() in ("mxnet"), reason=f"MXNet bug in advanced indexing (Issue 18919).")
```

Comment:
I've gotten the other TT methods to work, but this one has an indexing problem where `core` is first created. I believe it's the same issue as this Issue on the MXNet repository.